alpha/apps/kaltura/lib/kPlaybackContextDataHelper.php

```diff
@@ -236,7 +238,9 @@ private function createFlavorsMapping(entry $dbEntry)
 			switch($servePriority)
 			{
 				case StorageProfile::STORAGE_SERVE_PRIORITY_KALTURA_ONLY:
-					$c->addAnd(FileSyncPeer::FILE_TYPE, FileSync::FILE_SYNC_FILE_TYPE_URL, Criteria::NOT_EQUAL);
+					$c1 = $c->getNewCriterion(FileSyncPeer::FILE_TYPE, FileSync::FILE_SYNC_FILE_TYPE_URL, Criteria::NOT_EQUAL);
```

Comment:
Lets move to a function - getKalturaOnlyFileSyncs();
It should only access and get the special storage id's